tutorials/cp-starter-integration-cpi-onboard-subscribe/cp-starter-integration-cpi-onboard-subscribe.md

```diff
@@ -1,79 +1,98 @@
 ---
 title: Set Up Your SAP Cloud Platform Integration Tenant
-description: Subscribe to the Process Integration service, assign required roles, and use the self-service application to provision your own SAP Cloud Platform integration tenant.
+description: Subscribe to the Process Integration service, assign required roles, and use the self-service application to provision your own SAP Cloud Platform Integration tenant.
 auto_validation: true
 time: 15
 tags: [ tutorial>beginner, products>sap-cloud-platform, products>sap-cloud-platform-connectivity, products>sap-cloud-platform-for-the-cloud-foundry-environment]
 primary_tag: products>sap-cloud-platform-integration-for-process-services
+author_name: VIKRAM KULKARNI
+author_profile: https://github.com/Vikramkulkarni01
 ---
 
 ## Details
 ### You will learn
   - How to subscribe to the Process Integration service
   - How to set up your own SAP Cloud Platform Integration tenant
 
-To provision your SAP Cloud Platform Integration tenant, you must first subscribe to the process integration service, assign the necessary roles to your user, and then provision your SAP Cloud Platform Integration tenant.
+###Prerequisites
+To provision your SAP Cloud Platform Integration tenant, you must first subscribe to the Process Integration service, assign the necessary roles to your user, and then provision your SAP Cloud Platform Integration tenant.
```

Comment:
How will the developer know how to fulfilled prerequisites?

Reply:
Hello @thecodester, this prerequisite has been added by the previous author and I can see that it still exists in the prod tutorial. I definitely don't know the rationale behind this. 

let me check with @gauthamkrishna-sap  on this.

Reply:
Borders have been added to the below images. 
4.3.create.new.instance.png
4.4.specify-json-instance.png

Reply:
Removed prerequisites.